app/main.py

```diff
@@ -0,0 +1,68 @@
+from PyQt6.uic import loadUi
+from PyQt6.QtWidgets import QMainWindow, QApplication, QFileDialog
+from PyQt6.QtGui import QTextCursor
+
+import sys
```

Comment:
Please arrange imports in order according with [PEP8 import style guide](https://peps.python.org/pep-0008/#imports)